src/forms/BoundSelectAndTextField.test.tsx

```diff
@@ -0,0 +1,69 @@
+import { ObjectConfig, required, useFormState } from "@homebound/form-state";
+import { click } from "@homebound/rtl-utils";
+import { fireEvent } from "@testing-library/react";
+import { Observer } from "mobx-react";
+import { BoundSelectAndTextField } from "src";
+import { ScheduleTypes } from "src/components/Filters/testDomain";
+import { render, type } from "src/utils/rtl";
+
+describe("BoundSelectAndTextField", () => {
+  it("renders and updates values", async () => {
```

Comment:
**Question:** is there any value to testing that clearing works when rendered as clearable? Might be redundant based on other coverage.

Reply:
Good question. I don't think that test would be needed. I agree that it would be redundant based on the other test coverage.